### PR TITLE
[CDAP-7886] Adds run time arguments for preview in UI

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds run time arguments container to preview mode in hydrator studio

### CODE REVIEW

File: cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js [hunk 1]
```diff
@@ -208,7 +220,8 @@ class HydratorPlusPlusTopPanelCtrl{
       startStages: [],
       endStages: [],
       // useSinks: [], // we are not using sinks for now
-      numOfRecords: 25
+      numOfRecords: 25,
```
Comment (yaojiefeng) [inline cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js]: I think we do not need this for preview right now. The numOfRecords is actually not part of preview config, it is part of ETLconfig in the backend

**Comment by yaojiefeng** [comment]
LGTM

File: cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js [hunk 2]
```diff
@@ -207,8 +219,7 @@ class HydratorPlusPlusTopPanelCtrl{
     let previewConfig = {
       startStages: [],
       endStages: [],
-      // useSinks: [], // we are not using sinks for now
-      numOfRecords: 25
+      runTimeArguments: this.macrosMap
```
Comment (elfenheart) [inline cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js:222]: if `this.macrosMap` is empty, is the backend OK with empty runTimeArguments?

Comment (ajainarayanan) [reply cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js:222]: Yes. I believe so. @yaojiefeng Is that correct?

Comment (yaojiefeng) [reply cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js:222]: Yes, an empty runtimeArgument is fine.

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**